### DISCUSSION TITLE
IOS sync Folder Status is “out of sync”

### DISCUSSION THREAD

**Discussion by JacianLiu** [MobiusSync/MobiusSync#190] [Q&A]
Execute for a moment and then prompt out of sync

![IMG_7611](https://github.com/user-attachments/assets/ee80bf6d-f52c-4eee-b5c1-9a8bfc7e73a7)

# Log：

![IMG_7608](https://github.com/user-attachments/assets/35872d97-8c56-41c4-b71f-440867af0a9a)

# Device Info

iPhone 15 Pro MAX
iOS version: 17.3.1
iOS Permissions
![IMG_7612](https://github.com/user-attachments/assets/59224062-fda6-4d11-8895-13334ac8df35)

# I look forward to your reply, it is very important to me

**Comment by SimonPickup** [comment]
Is this Möbius Sync Pro? Or have you purchased unlimited file sync? The folder is greater than the free maximum of 20MB.

**Comment by JacianLiu** [comment]
> Is this Möbius Sync Pro? Or have you purchased unlimited file sync? The folder is greater than the free maximum of 20MB.

yes , this is Mobius Sync Pro.

**Comment by JacianLiu** [comment]
> Is this Möbius Sync Pro? Or have you purchased unlimited file sync? The folder is greater than the free maximum of 20MB.

What information do you need to quickly locate the problem?

**Comment by SimonPickup** [comment]
You may want to try some of the tips here: https://forum.obsidian.md/t/sync-mac-pc-and-ios-using-syncthing-mobius-sync/72022
